src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -197,6 +248,12 @@
       <OutputFolderForTestDependencies>$(BinDir)/TestDependencies</OutputFolderForTestDependencies>
     </PropertyGroup>
 
+    <!-- For UAP we need to copy the generated resources.pri containing the framework resources to the test folder -->
+    <Copy Condition="'$(BuildingUAPVertical)' == 'true' AND '$(TestProjectNeedsModifiedPriFile)' != 'true'"
+          SourceFiles="$(CommonPriFile)"
```

Comment:
Should we eliminate this special case and handle it directly in MakeTestSpecificResourcesPriFile so after that target we only have a resources.pri file. Either a copy of the common one or the common plus the test pri.